texttospeech/synthesize_file/synthesize_file_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+const TEXT_FILE = "../resources/hello.txt"
```

Comment:
Would you mind moving these to a `const` group?

```
const (
  textFile   = "../resources/hello.txt"
  ssmlFile   = "../resources/hello.ssml"
  outputFile = "output.mp3"
)
```

Reply:
Sure, changed